synapse/axon.py

```diff
@@ -133,6 +133,8 @@ def _fireAxonIden(self, iden):
     def _onAxonHostFini(self):
         for axon in list(self.axons.values()):
             axon.fini()
+        if self.axonbus is not None:
```

Comment:
I'd rather we go with
```
self.onfini( self.axonbus.fini )
```
in the section where we create the axonbus ( similar to Axon ) for consistency.

Reply:
done